releng/test_config.yaml

```diff
@@ -316,7 +278,7 @@ testSuites:
     - --timeout=180m
     - --test_args=--ginkgo.focus=\[Feature:(ExternalTrafficLocalOnly)\] --minStartupPods=8
     cluster: k8s-infra-prow-build
-  betaapis: # copied from "default" for now.  Eventually the args should specifically choose tests tagged for Beta, but this is to get us started.
+  betaapis: # copied from "default".  Tests that require beta APIs should use discovery in the e2e test to decide whether or not to skip the test.
```

Comment:
@liggitt I think this is the cleanest way for us write e2e tests that are easy to use for us and for downstream consumers.  The alternatives of `[Beta]` tag in the test name seems confusing since beta APIs are not the same as beta featuregates or beta fields.

Reply:
That seems reasonable for beta endpoints. I agree that discovering advertised things from the cluster under test is user friendly.

The risk that occurs to me is that something about the discovery check in the test is bugged and we skip tests we thought we were running